### PR TITLE
initialize RPC input and output entries when augmented

### PR DESCRIPTION
**Pull request by karimra**
This PR fixes the issue raised in #247

### CODE REVIEW

File: pkg/yang/entry.go [hunk 1]
```diff
@@ -1344,8 +1344,20 @@ func (e *Entry) Find(name string) *Entry {
 			_, part = getPrefix(part)
 			switch part {
 			case "input":
+				if e.RPC.Input == nil {
+					e.RPC.Input = &Entry{
+						Name: "input",
+						Dir:  make(map[string]*Entry),
+					}
```
Comment (wenovus) [inline pkg/yang/entry.go:1351]: Kind: InputEntry

File: pkg/yang/entry.go [hunk 2]
```diff
@@ -1344,8 +1344,20 @@ func (e *Entry) Find(name string) *Entry {
...
+				}
 				e = e.RPC.Input
 			case "output":
+				if e.RPC.Output == nil {
+					e.RPC.Output = &Entry{
+						Name: "output",
+						Dir:  make(map[string]*Entry),
+					}
```
Comment (wenovus) [inline pkg/yang/entry.go:1359]: Kind: OutputEntry

**Review by wenovus — CHANGES_REQUESTED** [review]
Sorry for missing replying on the issue. This PR LGTM, one small suggestion.

File: pkg/yang/entry.go (hunk 1 above)
Comment (karimra) [reply pkg/yang/entry.go:1352]: done

File: pkg/yang/entry.go (hunk 2 above)
Comment (karimra) [reply pkg/yang/entry.go:1361]: done

**Review by wenovus — APPROVED** [review]

**Pull request merged**